test/jasmine/tests/hover_spikeline_test.js

```diff
@@ -33,6 +33,14 @@ describe('spikeline', function() {
             Lib.clearThrottle();
         }
 
+        function _set_hovermode(hovermode) {
+            gd._fullLayout.hovermode = hovermode;
+        }
+
+        function _set_spikedistance(spikedistance) {
+            gd._fullLayout.spikedistance = spikedistance;
```

Comment:
Could you use `return Plotly.relayout(gd, 'spikedistance', spikedistance)` instead? This would mimic a more real-world situation.

Reply:
Thanks! Done.